client/payment-methods/capability-request/capability-request-dismiss-modal.tsx

```diff
@@ -0,0 +1,53 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __, sprintf } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { DismissConfirmationModalProps } from './types';
+
+/**
+ * Internal dependencies
+ */
+import ConfirmationModal from 'wcpay/components/confirmation-modal';
+
+const DismissConfirmationModal: React.FC< DismissConfirmationModalProps > = ( {
+	onClose,
+	onSubmit,
+	label,
+} ): JSX.Element => {
+	const buttonContent = (
+		<>
+			<Button isSecondary onClick={ onClose }>
+				{ __( 'Cancel', 'woocommerce-payments' ) }
+			</Button>
+			<Button isPrimary onClick={ onSubmit }>
+				{ __( 'Yes, continue', 'woocommerce-payments' ) }
+			</Button>
+		</>
+	);
+
+	return (
+		<ConfirmationModal
+			title={ __( 'Remove', 'woocommerce-payments' ) + ' ' + label }
+			isDismissible={ false }
+			className="dismiss-confirmation-modal"
+			onRequestClose={ onClose }
+			actions={ buttonContent }
+		>
+			<p>
+				{ sprintf(
+					/** translators: %s is the currency code, e.g. USD. */
+					__(
+						'Please confirme, by chooising "Yes, continue" you will remove the option to add the ' +
```

Comment:
a typo here `confirme`

Reply:
There's another typo `chooising`

Reply:
Here's my attempt at a refined (and hopefully grammatically correct) message:

> Choosing to continue will remove the option to accept JCB cards from your customers. The option to enable JCB will not appear again.

Reply:
Thank you, the message is updated!